src/gateway/environments/jsdom-sixteen/jsdom-sixteen-resource-loader.js

```diff
@@ -70,7 +71,7 @@ export class JSDOMSixteenResourceLoader extends ResourceLoader {
         super();
 
         if (renderAPI == null) {
-            throw new Error("Must provide render API.");
+            throw new KAError("Must provide render API.", Errors.Internal);
```

Comment:
InvalidInput here too?

Reply:
Again, this is not due to invalid input to the service, only in coding, so I don't believe it fits InvalidInput